src/vochain/vochain.proto

```diff
@@ -16,18 +16,33 @@ message VoteEnvelope {
         repeated uint32 encryptionKeyIndexes = 6; // On encrypted votes, contains the (sorted) indexes of the keys used to encrypt
 }
 
+enum CensusType {
+	Unknown = 0;
+	Graviton = 1;
+	Iden3 = 2;
+	EthereumStorage = 3;
+	EthereumAccount = 4;
+	CA = 5;
+	ArboPoseidon = 6;
+	ArboBlake2b = 7;
+}
+
 message Proof {
         oneof payload {
-                // Proof used for signed envelopes
-                ProofGraviton graviton = 1;
-                // Proof used for anonymous votes
-                ProofIden3 iden3 = 2;
-                // Proof used on EVM census based processes
-                ProofEthereumStorage ethereumStorage = 3;
-                // Proof used by oracles to update the census of EVM processes
-                ProofEthereumAccount ethereumAccount = 4;
-                // Proof Certification Authority
-                ProofCA ca = 5;
+		// Proof used for signed envelopes
```

Comment:
Why this different identation? Is this because we have different configs on IDEs?

Reply:
My fault, fixed in the amend